website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Exports"
+description: "Use Exports to materialize tables to the data platform on a schedule."
+sidebar_label: "Materialize queries with Exports"
+---
+
+Exports in the dbt Semantic Layer extends the [saved queries](/docs/build/saved-queries) functionality:
+
+- Exports enable you to materialize and automate these queries within your data platform.
+- Exports uses the dbt Cloud job scheduler to execute saved queries for reliable and fast data reporting.
+
+While saved queries are a way to save and reuse commonly used queries in MetricFlow, Exports takes this a step further by using the MetricFlow server to materialize these queries as data tables of views using the dbt Cloud job scheduler.
+
+| Feature    | Exports  | Saved queries  |
+|-----------|-----------|----------------|
+| **Availability**    | Available to dbt Cloud users on [Team or Enterprise](https://www.getdbt.com/pricing/) plan on  on dbt versions 1.7 or higher.| Available in both dbt Core and dbt Cloud.     |
+| **Purpose**         | To schedule and run saved queries as part of [dbt's job scheduler](/docs/deploy/job-scheduler). | To define and manage common Semantic Layer queries in YAML, which includes metrics and dimensions.   |
+| **Usage**           | Automatically runs saved queries and materializes them within your data platform. <br /><br />**Example**: Creating a weekly aggregated table for active user metrics, automatically updated and stored in the data platform.  | Used for organizing and reusing common MetricFlow queries within dbt projects.<br /><br />**Example**: Standardizing a frequently used revenue calculation across multiple reports. | For materializing query results in the data platform. |
+| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's job scheduler. | Integrated into dbt DAG and managed alongside other dbt nodes. |
+| **Configuration**   | Configured within dbt Cloud environment and job scheduler settings. | Defined in YAML format within dbt project files.   |
+
+## Define exports
+
+Exports are essentially saved queries that you can schedule and execute using [dbt's job scheduler](/docs/deploy/job-scheduler) capabilities. Exports include the following materialization types: CAN A USER ALSO USE API OR CI JOBS? WHAT IF THEY USE ANOTHER ORCHESTRATOR?
+
```

Comment:
CAN A USER ALSO USE API OR CI JOBS? WHAT IF THEY USE ANOTHER ORCHESTRATOR?

Reply:
Yea they should be able to call the cloud API to run an export job.

Reply:
I don't think that functionality has been built yet. We are just using internal APIs to execute the Export, so we haven't exposed anything to the user yet.

Reply:
ok so i should leave anything api-related out?

Reply:
meaning the only way to run exports is via the dbt cloud jbo scheduler?

Reply:
Just the Job Scheduler and the Cloud CLI, no API functionality yet.